README.md

```diff
@@ -2,14 +2,14 @@
 
 StreetComplete is an easy to use editor of OpenStreetMap data available for Android. It can be used without any OpenStreetMap-specific knowledge. It asks simple questions, with answers directly used to edit and improve OpenStreetMap data. The app is aimed at users who do not know anything about OSM tagging schemes but still want to contribute to OpenStreetMap.
 
-StreetComplete automatically looks for nearby places where a survey is needed and shows those as quest markers on its map. Each of these quests can then be solved on site by answering a simple question. For example, tapping on a marker may show the question "What is the name of this road?", with a text field to answer it.
+StreetComplete automatically looks for nearby places where a survey is needed and shows them as quest markers on its map. Each of these quests can then be solved on site by answering a simple question. For example, tapping on a marker may show a question "What is the name of this road?", with a text field to answer it.
```

Comment:
Revert this and I'll merge it

Reply:
Including the -> a change?

Reply:
The those -> them change is good. The the -> a change is bad, as it refers to one specific wording and not a more vague example or multiple specific examples.

Reply:
K has goldfndr said